src/parser/csql_grammar.y

```diff
@@ -5505,6 +5508,11 @@ class_name_with_server_name
                 PT_NAME_INFO_CLEAR_FLAG($1, PT_NAME_INFO_USER_SPECIFIED);
                 SET_CONTAINER_2 (ctn, $1, $3);
                 $$ = ctn;
+		if (is_in_create_trigger)
+		  {
+		    PT_ERROR(this_parser, $1, "check syntax: Not supported @!!!");
```

Comment:
Shouldn't the error message be modified to be more appropriate?

Reply:
Fixed.